app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] You have a typo here. You have written `:database_authenticatable` where it should be `:database_authenticable`

Reply:
Hello thank you for the review, I did check it once again, and the correct one is "authenticatable". Even in different frameworks like Laravel, it is the same.
Here are some sources: 
https://www.yourdictionary.com/authenticatable
https://en.wiktionary.org/wiki/authenticatable

Just to make sure, I did try your suggestion, but this returned with a name error.
Also, thank you once again for your time :)

Reply:
Hei @Ewoskyx, I just double checked and you are absolutely right. Sorry for the confusion